src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -23,50 +23,54 @@
+    public static BlockHutStonemason             blockHutStonemason;
+    public static BlockHutFarmer                 blockHutFarmer;
+    public static BlockHutFisherman              blockHutFisherman;
+    public static BlockSubstitution              blockSubstitution;
+    public static BlockBarracksTowerSubstitution blockBarracksTowerSubstitution;
+    public static BlockSolidSubstitution         blockSolidSubstitution;
+    public static BlockHutField                  blockHutField;
+    public static BlockHutGuardTower             blockHutGuardTower;
+    public static BlockHutWareHouse              blockHutWareHouse;
+    public static BlockHutShepherd               blockHutShepherd;
+    public static BlockHutCowboy                 blockHutCowboy;
+    public static BlockHutSwineHerder            blockHutSwineHerder;
+    public static BlockHutChickenHerder          blockHutChickenHerder;
+    public static BlockHutBarracks               blockHutBarracks;
+    public static BlockHutBarracksTower          blockHutBarracksTower;
+    public static BlockHutCook                   blockHutCook;
+    public static BlockHutSmeltery               blockHutSmeltery;
 
     /**
      * Utility blocks.
      */
-    public static BlockConstructionTape       blockConstructionTape;
-    public static BlockMinecoloniesRack       blockRack;
-    public static BlockWaypoint               blockWayPoint;
-    public static BlockInfoPoster             blockInfoPoster;
-    public static BlockPaperwall              blockPaperWall;
-    public static BlockShingle                blockShingleOak;
-    public static BlockShingle                blockShingleBirch;
-    public static BlockShingle                blockShingleJungle;
-    public static BlockShingle                blockShingleSpruce;
-    public static BlockShingle                blockShingleDarkOak;
-    public static BlockShingle                blockShingleAcacia;
-    public static BlockShingleSlab            blockShingleSlab;
-    public static MultiBlock                  multiBlock;
-
+    public static BlockConstructionTape blockConstructionTape;
+    public static BlockMinecoloniesRack blockRack;
+    public static BlockWaypoint         blockWayPoint;
+    public static BlockInfoPoster       blockInfoPoster;
+    public static BlockPaperwall        blockPaperWall;
+    public static BlockShingle          blockShingleOak;
+    public static BlockShingle          blockShingleBirch;
+    public static BlockShingle          blockShingleJungle;
+    public static BlockShingle          blockShingleSpruce;
+    public static BlockShingle          blockShingleDarkOak;
+    public static BlockShingle          blockShingleAcacia;
+    public static BlockShingleSlab      blockShingleSlab;
+    public static MultiBlock            multiBlock;
 
+    public static List<BlockTimberFrame> getTimberFrames()
+    {
+        return timberFrames;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "timberFrames". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)